portal/templates/gil/base.html

```diff
@@ -150,16 +151,21 @@ <h3 class="module__head">Everyone has a part to play in improving the prostate c
         <ul class="nav-list">
           <li class="nav-list__item"><a href="{{PORTAL}}/">Home</a></li>
           <li class="nav-list__item"><a href="{{PORTAL}}/about">About</a></li>
-          <li class="nav-list__item"><a class="decision-support-link" href="{{PORTAL}}/decision-support">Decision Support</a></li>
-          <li class="nav-list__item"><a class="symptom-tracker-link" href="{{PORTAL}}/symptom-tracker">Symptom Tracker</a></li>
+          <li class="nav-list__item"><a class="decision-support-link" href="{{PORTAL}}{{url_for('portal.decision_support')}}">Decision Support</a></li>
```

Comment:
Could you wrap these links in translation calls? Also thank you for using `url_for()`! :)